src/PrestaShopBundle/Service/DataProvider/Admin/CategoriesProvider.php

```diff
@@ -51,6 +51,11 @@ class CategoriesProvider
      */
     private $categories;
 
```

Comment:
Could you add some unit tests for these changes in `tests/Unit/PrestaShopBundle/Service/DataProvider/Admin/CategoriesProviderTest.php` ?